.github/ISSUE_TEMPLATE/product_update_facdb.md

```diff
@@ -0,0 +1,146 @@
+- [ ] moeo_socialservicesitelocations https://data.cityofnewyork.us/City-Government/Verified-Locations-for-NYC-City-Funded-Social-Serv/2bvn-ky2h
+- [ ] nycha_communitycenters https://data.cityofnewyork.us/Social-Services/Directory-of-NYCHA-Community-Facilities/crns-fw6u
+- [ ] nycha_policeservice https://data.cityofnewyork.us/Housing-Development/NYCHA-PSA-Police-Service-Areas-/72wx-vdjr
+  NOTE: this data is shown as a map on the website.
+- [ ] nysdec_solidwaste https://data.ny.gov/Energy-Environment/Solid-Waste-Management-Facilities/2fni-raj8
+- [ ] nysdoh_healthfacilities https://health.data.ny.gov/Health/Health-Facility-General-Information/vn5v-hh5r
+- [ ] nysdoh_nursinghomes https://health.data.ny.gov/Health/Nursing-Home-Weekly-Bed-Census-Last-Submission/izta-vnpq
+- [ ] nysed_nonpublicenrollment http://www.p12.nysed.gov/irs/statistics/nonpublic/
+- [ ] nysomh_mentalhealth https://data.ny.gov/Human-Services/Local-Mental-Health-Programs/6nvr-tbv8
+- [ ] nysopwdd_providers https://data.ny.gov/Human-Services/Directory-of-Developmental-Disabilities-Service-Pr/ieqx-cqyk
+- [ ] nysparks_historicplaces https://data.ny.gov/Recreation/National-Register-of-Historic-Places/iisn-hnyv
+- [ ] nysparks_parks https://data.ny.gov/Recreation/State-Park-Facility-Points/9uuk-x7vh
+- [ ] qpl_libraries https://data.cityofnewyork.us/Education/Queens-Library-Branches/kh3d-xhq7
+- [ ] sbs_workforce1 https://data.cityofnewyork.us/dataset/Center-Service-Locations/6smc-7mk6
+- [ ] usdot_airports https://geodata.bts.gov/datasets/aviation-facilities/explore?location=50.755910%2C-117.686932%2C22.58&showTable=true
+  Use data-library to extract and archive data. Note, data-library uses a different link to extract the data. This link is provided as a reference for last date updated.  
+- [ ] usdot_ports https://data-usdot.opendata.arcgis.com/datasets/usdot::docks/about
+  Use data-library to extract and archive data
+
+## Manually check data for updates
+
+The source datasets don't currently provide an API or export option on their websites. The data engineering team initially created the datasets 
+and continues to maintain them internally, relying on the information available on the websites. Also, the source data websites don't report 
+date updated as neatly as the open datasets, have to look at data itself. 
+
+- [ ] fbop_corrections https://www.bop.gov/locations/list.jsp
+  When searching by state, there should be 5 NY prisons, 3 of which are in NYC (Brooklyn/New York)
+- [ ] nycdoc_corrections  https://www1.nyc.gov/site/doc/about/facilities-locations.page
+  Source: NYCDOC locations directory
+- [ ] nycourts_courts http://www.nycourts.gov/courts/nyc/criminal/generalinfo.shtml#BRONX_COUNTY
+- [ ] nysdoccs_corrections https://doccs.ny.gov/find-facility
+  Hand check for 1 facility in queens, 1 facility in Manhattan, 0 in the other 3 boros. Only look at the correctional facility locations, not the offices.
+
+## Manual download
```

Comment:
If you have bandwidth, would be cool to replace any documentation (or just put a TODO) to indicate that we should put these files in edm-recipes/inbox on S3, and change the `library` template path to point there, e.g. what we do [here](https://github.com/NYCPlanning/data-engineering/blob/cd5c71319b0c093154baf8885b3badf6709451fd/dcpy/library/templates/panynj_jfk_65db.yml#L6)

Reply:
Great suggestion @alexrichey!

Reply:
I added a todo tag. Since it's a bit out of scope of this pr, I will do it separately